### PR TITLE
second street address to ind/org

### LINKED ISSUE #79: Add another street_address line to Individuals and Organizations
We want:
street1
street2
city
state
zip

### PR DESCRIPTION
**Pull request by RachellCalhoun**
fixes #79

### CODE REVIEW

File: portal/contacts/models.py [hunk 1]
```diff
@@ -8,6 +8,8 @@ class Meta:
         abstract = True
 
     street_address = models.CharField(max_length=100, blank=True, null=True)
+    second_street_address = models.CharField(
+        max_length=100, blank=True, null=True)
```
Comment (jacebrowning) [inline portal/contacts/models.py]: My understating of #79 is that they're simply looking for an optional second line to addresses. `second_street_address` kind of sounds like they might have two addresses.

Would it make sense for these fields to simply be `street_address_line_1` and `street_address_line_2`?

Comment (RachellCalhoun) [reply portal/contacts/models.py]: Ok. I'll change it. Naming is hard. But you're a rockstar at it! Ha thanks!

Comment (jacebrowning) [reply portal/contacts/models.py]: This _might_ require a field name to be corrected in `importdata.py`. I don't anticipate we'll ever run that again, but it doesn't hurt to keep that in sync.

**Comment by oneblackcrayon** [comment]
Closed as it is outdated.

**Pull request closed without merging**